.github/workflows/deploy-scripts-internally.yaml

```
@@ -34,7 +34,14 @@ jobs:
 
                     cd /tmp/nango-temp/nango-integrations
                     npm install zod soap botbuilder
-                    nango admin:deploy-internal template-$integration
+
+                    response=$(nango admin:deploy-internal template-$integration)
+
+                    echo "$response" | jq '.error.code' | grep -q "unknown_provider_config"
+                    if [ $? -eq 0 ]; then
+                      echo "Error: Unknown provider config for $integration"
```

```suggestion
                      echo "Warning: Unknown provider config for $integration"
```
since we are not failing